src/core/tsi/s2a/handshaker/s2a_tsi_handshaker.cc

```diff
@@ -148,16 +297,69 @@ tsi_result s2a_tsi_handshaker_create(
 
 static tsi_result s2a_handshaker_result_extract_peer(
```

Comment:
Please also add the TSI peer property for security level. The https://github.com/grpc/grpc/pull/21215/files is merged now.

Reply:
If it's ok with you, I intend to do this in a separate PR. Adding the security level peer property necessitates modifying the security connector, which is already being modified in an open PR (PR #20 - I'm waiting on approval by @cesarghali). WDYT?

To prevent any breakages, I've held off merging the `s2a-master` branch with the new version of `master` that contains the changes from your PR.

Reply:
Doing it in a separate PR SGTM.